### PR TITLE
Merge from master; 5.0.0-beta.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
+- Initial beta release.
+- Added support for `5.0.0-beta.*` versions of SKY UX component libraries. [#269](https://github.com/blackbaud/skyux-flyout/pull/269)
+- Fixed `SkyFlyoutService` to work within lazy-loaded feature modules. [#115](https://github.com/blackbaud/skyux-flyout/pull/115)
+
 # 5.0.0-alpha.0 (2021-05-21)
 
+- Added support for `@angular/core@^12`. [#114](https://github.com/blackbaud/skyux-flyout/pull/114)
 - Removed `BrowserAnimationsModule` from the `imports` section of `SkyFlyoutModule` to support lazy-loading. Consumers of `SkyFlyoutModule` must now import `BrowserAnimationsModule` into their application's root module. [#114](https://github.com/blackbaud/skyux-flyout/pull/114)
 
+# 4.2.0 (2021-06-10)
```
Comment (Blackbaud-SteveBrush) [inline CHANGELOG.md:12]: Should this entry be placed above the `5.0.0-alpha.0` entry?

File: src/app/public/modules/flyout/flyout.component.spec.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ import {
 import {
   Router
 } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/flyout/flyout.component.spec.ts:16]: Import code style.

Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/flyout/flyout.component.spec.ts]: Fixed.

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Done.

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**